### PR TITLE
Create navbar and button for submission page on index.html and style via CSS

### PR DESCRIPTION
**Pull request by RafikaAT**
:D

### CODE REVIEW

File: client/assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    text-align: center;
+    color: #FFFFFF;
+    font-family: 'Roboto', sans-serif;
+}
+
+.button-centering-container {
+    height: 200px;
+    border: 3px solid green;
+}
+
```
Comment (akashkhambay) [inline client/assets/css/style.css:59]: Nice work

File: client/index.html [hunk 2]
```diff
@@ -4,9 +4,27 @@
...
+				<h5><a class="navbar-anchors" href="#">Movies</a></h5>
+			</div>
+			
+		</nav>
+		<br>
+		<div id="button-centering-container">
+			<button>Post your journal entry!</button>
+		</div>
+		
+	</body>
```
Comment (akashkhambay) [inline client/index.html:24]: I like this, well done

File: client/index.html [hunk 3]
```diff
@@ -4,9 +4,27 @@
...
 		<script defer src="./assets/bundle.js"></script>
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+		<link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
 		<link rel="stylesheet" href="assets/css/style.css" />
 	</head>
-	<body></body>
+	<body>
+		<nav>
+			<h4 id="h4" class="navbar-grid-item">JOURNEL</h4>
```
Comment (Velocima) [inline client/index.html]: An H4 with an id of h4 is superfluous.
Consider using an h1 or h2 as a more semantic choice.

File: client/index.html [hunk 4]
```diff
@@ -4,9 +4,27 @@
...
+			<div class="navbar-grid-item" id='navbar-grid'>
+				<h5><a class="navbar-anchors" href="#">Anime</a></h5>
```
Comment (Velocima) [inline client/index.html]: class here is superfluous. can select with `nav div h5 a`

File: client/index.html [hunk 5]
```diff
@@ -4,9 +4,27 @@
...
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+		<link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
 		<link rel="stylesheet" href="assets/css/style.css" />
 	</head>
-	<body></body>
+	<body>
+		<nav>
+			<h4 id="h4" class="navbar-grid-item">JOURNEL</h4>
+			<div class="navbar-grid-item" id='navbar-grid'>
```
Comment (Velocima) [inline client/index.html]: Having an id an a class here is potentially redundant? 
Consider the level of specificity required

File: client/index.html [hunk 6]
```diff
@@ -4,9 +4,27 @@
...
+				<h5><a class="navbar-anchors" href="#">Anime</a></h5>
+				<h5><a class="navbar-anchors" href="#">Food</a></h5>
+				<h5><a class="navbar-anchors" href="#">Movies</a></h5>
+			</div>
+			
+		</nav>
+		<br>
+		<div id="button-centering-container">
```
Comment (Velocima) [inline client/index.html]: container may be unnecessary?

File: client/assets/css/style.css [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+body {
+    background-image: url('images/homepage-image.png');
+    background-size: cover;
+    background-position: center;  
+    background-repeat: no-repeat;
+    min-height: 100vh;
+    background-attachment: fixed;
+    margin: 0;
+}
```
Comment (Velocima) [inline client/assets/css/style.css]: When applying multiple background properties you can use the combined background property.
Definitely up to you whether you prefer declaring them individually or on one property.
Check out [this link](https://www.w3schools.com/cssref/css3_pr_background.asp) for more info.

File: client/assets/css/style.css [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+
+nav {
+    display: grid;
+    background-color: #2C53DA;
+    grid-template-columns: 30% 70%;
+    border: 1px solid rgba(0, 0, 0, 0.8);
+    width: 100%;
+}
+
+#h4 {
```
Comment (Velocima) [inline client/assets/css/style.css]: Using the id selector is quite specific here.  Can you use the element selector instead?  Would a class selector be more appropriate?

Comment (RafikaAT) [reply client/assets/css/style.css]: I'm only assigning properties to this specific part of the html body, so I thought class would be needless. Also, I need to prioritize the properties assigned here over properties assigned to the two containers making up the outermost grid of the navbar. Using the element selector assigns the wrong properties to this tag.
Screenshot 2021-07-05 at 19 43 41

File: client/index.html (hunk 5 above)
Comment (RafikaAT) [reply client/index.html]: See earlier response to the h4 id, por favor :)

File: client/assets/css/style.css (hunk 1 above)
Comment (RafikaAT) [reply client/assets/css/style.css:59]: Thanks :)

File: client/index.html (hunk 4 above)
Comment (RafikaAT) [reply client/index.html]: I attempted this and the browser did not like that:
Screenshot 2021-07-05 at 19 51 19

File: client/index.html (hunk 3 above)
Comment (RafikaAT) [reply client/index.html]: So an id of h1 for a h4 tag?

**Pull request closed without merging**